pkg/prometheus/prometheus.go

```diff
@@ -0,0 +1,74 @@
+package prometheus
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/prometheus/client_golang/api"
+	v1 "github.com/prometheus/client_golang/api/prometheus/v1"
```

Comment:
```suggestion
	promv1 "github.com/prometheus/client_golang/api/prometheus/v1"
```
so that it is easier to identify we are calling prometheus functions.